test/hardware/AutomatedHWTests/TestFreeRTOSQueue.c

```diff
@@ -0,0 +1,116 @@
+#define SOURCE_FILE "FREERTOS-HWTEST"
+
+#include "Common.h"
+#include "EnV5HwController.h"
+#include "FreeRtosQueueWrapper.h"
+#include "FreeRtosTaskWrapper.h"
+#include "stdlib.h"
+#include "string.h"
+#include "unity.h"
+
+#include "pico/bootrom.h"
+#include "pico/stdio_usb.h"
+
+#define QUEUE_SIZE 5
+
+typedef struct message {
+    uint16_t messageId;
+    char content[15];
+} message_t;
+
+queue_t messageQueue;
+char staticMessage[] = "STATIC MSG";
+char dynamicMessage[] = "DYNAMIC MSG";
+
+void init(void) {
+    stdio_init_all();
+    while ((!stdio_usb_connected())) {}
+    env5HwControllerInit();
+}
+
+void testCreateQueue(void) {
+    TEST_ASSERT_NOT_NULL(messageQueue);
+}
+
+// tests basic message exchange
+void testMessageExchange() {
+    message_t sentMessage = {.messageId = 1};
+    strcpy(sentMessage.content, staticMessage);
+
+    TEST_ASSERT_TRUE(freeRtosQueueWrapperPush(messageQueue, &sentMessage));
+
+    message_t receivedMessage;
+    TEST_ASSERT_TRUE(freeRtosQueueWrapperPop(messageQueue, &receivedMessage));
+    TEST_ASSERT_EQUAL_UINT16(1, receivedMessage.messageId);
+    TEST_ASSERT_EQUAL_STRING(staticMessage, receivedMessage.content);
+}
+
+// tests if message content is dynamically changed before pushing to queue
+void testDynamicMessageReplacement(void) {
```

Comment:
I suggest removing the comment and calling the fn smth like
`testMessageContentDynamicallyChangedBeforeEnqueueing`.

Same goes for the other fns here that feature similar comments.